NSS-Day-Cohort-66/rare-client-dapper-pirates-1
# Kq create new category

I created a React module `Category.js` and added Tailwind styling. It handles all client interactions of adding a new `category`.

# Steps to test

1. Pull down the `kq-createNewCategory` branch and switch to it.
2. Ensure your back-end server is running.
3. within the branch at your terminal, run the command `npm start`
4. While in the browser, navigate via the navbar to the Category Manager page.
**5. Try adding the same tag, there should be a window alert error**
6. Try creating new tag. Once the create button is clicked the fields should clear and new tag added/repopulates on the list
